packages/client/hmi-client/src/types/ModelConfig.ts

```diff
@@ -0,0 +1,9 @@
+import { Model } from '@/types/Model';
+
+// This is temp as it will be replaced by types/Types when this is more finalized and moved to hmi-server
+export interface ModelConfig {
```

Comment:
If this is a new file just for this PR, is it worth it to just do the back end file and `@TSModel` it?

Reply:
Maybe for next week? This is in flux because
- TDS has no model-config yet, and
- Waiting for everyone's api/interface (TDS, sim-service, hmi-server) to catch up with the new model-representation

Reply:
I will gladly add this file and @TSModel it when TDS can create this